templates/security/login.html.twig

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Log in!{% endblock %}
+
+{% block content %}
+<div class="container">
+    <form method="post">
```

Comment:
Why don't you use symfony forms and twig form helpers?

Reply:
Didn't know they existed, I made the implementation